README.md

```diff
@@ -37,8 +37,8 @@ ENV.webhooks = {
         return {
           apiKey: <your-api-key>
         };
-      }
-      didActivate: true
+      },
+      didActivate: true,
```

Comment:
extra comma

Reply:
hi @marcoow! I don't understand what you're trying to say

Reply:
I don't think we use trailing commas in this repo

Reply:
A trailing comma in the readme doesn't seem like the worst of sins. It's removed now anyways.